packages/mdc-icon-button/README.md

```diff
@@ -140,6 +157,7 @@ cannot be disabled. Disabled icon buttons cannot be interacted with and have no
 CSS Class | Description
 --- | ---
 `mdc-icon-button` | Mandatory.
+`mdc-icon-button--on` | Used to indicate the toggle button icon that is the not-selected option.
```

Comment:
This doesn't sound right... This class is applied to the root element when the toggle should be in the "on" state, right?